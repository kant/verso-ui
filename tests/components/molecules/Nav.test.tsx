import { jsx } from '@emotion/react'
import renderer from 'react-test-renderer'
import { Nav } from 'src/components'

test('Snapshot Test: Contact Form', () => {
  const pagesArray = [
    {
      pathName: 'path/to/page/one',
      pageName: 'Page One',
    },
    {
      pathName: 'path/to/page/two',
      pageName: 'Page Two',
    },
  ]

  const component = renderer.create(
    <Nav pages={pagesArray}>Heading Level One</Nav>,
  )

  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
