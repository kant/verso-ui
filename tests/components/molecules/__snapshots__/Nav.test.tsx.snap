// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot Test: Contact Form 1`] = `
<nav
  className="css-23iqwz-Nav"
>
  <ul
    className="css-rk3qdn-Nav"
  >
    <li>
      <span
        className="css-r5wu0g"
        onClick={[Function]}
        onMouseEnter={[Function]}
      >
        <a
          className="css-8mhfhs-Text"
        >
          Page One
        </a>
      </span>
    </li>
    <li>
      <span
        className="css-r5wu0g"
        onClick={[Function]}
        onMouseEnter={[Function]}
      >
        <a
          className="css-8mhfhs-Text"
        >
          Page Two
        </a>
      </span>
    </li>
  </ul>
</nav>
`;
